.github/workflows/build_and_test.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       id: sync-branch
       run: |
         apache_spark_ref=`git rev-parse HEAD`
-        git fetch https://github.com/$GITHUB_REPOSITORY.git ${GITHUB_REF##*/}
+        git fetch https://github.com/$GITHUB_REPOSITORY.git ${GITHUB_REF#refs/heads/}
```

Comment:
Hi, @ueshin . If we need this, please make a separate JIRA issue for this.

Reply:
Sure. I tested it here to make sure it works. I'll revert the change and file a separate JIRA later. Thanks!

Reply:
Thanks for the confirmation! Sure, no problem.